plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/branch/BranchPresenter.java

```diff
@@ -216,6 +225,22 @@ protected void onSuccess(String result) {
                 projectExplorer.reloadChildren();
 
                 updateOpenedFiles();
+
+                //refresh project
+                projectService.getProject  (appContext.getWorkspace().getId(), path,
```

Comment:
Please reformat it. And it would better to use workspaceId variable her https://github.com/codenvy/che-plugins/blob/IDEX-3576/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/branch/BranchPresenter.java#L108